### PR TITLE
Fixes for edgeview

### PR DESCRIPTION
**Pull request by giggsoff**
Several fixes for edgeview:

1. Formatting fixes
2. Build process simplify and align versions
3. Check for received filename to not have relative path and be inside of expected directory
4. Show help without check for other options
5. Rename image in doc and Makefile to be aligned with docker org
6. Do not mount /persist/certs as we may not have it for live images
7. Add zfsmanager pubsub to the list

### CODE REVIEW

**Comment by eriknordmark** [comment]
@naiming-zededa can you review this?

**Comment by naiming-zededa** [comment]
> @naiming-zededa can you review this?

LGTM. Thanks @giggsoff

File: pkg/edgeview/src/copyfile.go [hunk 1]
```diff
@@ -184,18 +184,24 @@ func recvCopyFile(msg []byte, fstatus *fileCopyStatus, mtype int) {
...
 
 		_, err = os.Stat(fileCopyDir)
 		if err != nil {
 			sendCopyDone("file stat ", err)
 			return
 		}
-		fstatus.f, err = os.Create(fileCopyDir + fstatus.filename)
+		filePath := filepath.Join(fileCopyDir, fstatus.filename)
+		// check if we receive valid filename to avoid touch of files outside fileCopyDir
+		if !strings.HasPrefix(filePath, fileCopyDir) {
```
Comment (zededa-yuri) [inline pkg/edgeview/src/copyfile.go:195]: I guess this check still can be fooled by `/path_allowed_to_access/../no_no_no_you_should_not_be_here`

File: pkg/edgeview/README.md [hunk 2]
```diff
@@ -11,28 +11,31 @@ It contains an example of golang source for 'edgeview' websocket dispatcher runn
 ## Dockerfiles
```
Comment (zededa-yuri) [inline pkg/edgeview/README.md:11]: NIT
"edgeview: Fix image name to be aligned" -> "edgeview docs: Fix image name to be consistent"

File: pkg/edgeview/README.md [hunk 3]
```diff
@@ -34,7 +34,7 @@ edge-view-query [ -token <session-token> ] [ -debug ] [ -inst <instance-id> ] <q
  options:
```
Comment (zededa-yuri) [inline pkg/edgeview/README.md:34]: Would be great to explain in commit message why `zfsmamanger` paubsub is required here

File: pkg/edgeview/src/copyfile.go (hunk 1 above)
Comment (giggsoff) [reply pkg/edgeview/src/copyfile.go:195]: Join function do cleanup of the path inside.

Comment (zededa-yuri) [reply pkg/edgeview/src/copyfile.go:195]: oh, that's neat

File: pkg/edgeview/README.md (hunk 3 above)
Comment (giggsoff) [reply pkg/edgeview/README.md:34]: Good point, added

**Review by eriknordmark — APPROVED** [review]
LGTM

**Pull request merged**